go.mod

```diff
@@ -143,6 +143,7 @@ require (
 )
 
 replace (
+	github.com/go-git/go-git/v5 v5.4.2 => github.com/go-git/go-git/v5 v5.11.0
```

Comment:
have you tried `go mod tidy` on your branch after making this change? Generally. go.sum also gets updated when you do that (not necessarily though).

Reply:
No other changes were introduced after doing go mod tidy for this change..